### PR TITLE
bug 1523047. fix log archive link for operation namespaces

### PR DESCRIPTION
**Pull request by jcantrill**

### CODE REVIEW

File: app/scripts/services/logLinks.js [hunk 1]
```diff
@@ -73,7 +73,13 @@ angular.module('openshiftConsole')
       ].join(''));
 
 
-      var archiveUri = function(opts) {
+      var archiveUri = function(opts, prefix) {
+        var index = 'project.' + opts['namespace'] + '.' + opts['namespaceUid'] + '.*';
+        if(prefix) {
```
Comment (benjaminapetersen) [inline app/scripts/services/logLinks.js]: Maybe we can reduce some noise with `opts.namespace` & `opts.namespaceUid`, rather than array syntax?

**Comment by benjaminapetersen** [comment]
@jcantrill I can review, ideally I'd set it up & test, whats the best way to get things running these days?

File: app/scripts/directives/logViewer.js [hunk 2]
```diff
@@ -427,7 +427,7 @@ angular.module('openshiftConsole')
                                         podname: name,
                                         containername: $scope.options.container,
                                         backlink: URI.encode($window.location.href)
-                                      }))
+                                      }, $filter('annotation')($scope.context.project,'loggingDataPrefix')))
```
[the remark this answers is not in the document]
Comment (spadgett) [reply app/scripts/directives/logViewer.js]: Scratch that, I'm OK with it as a separate property since it's not passed to the template

**Comment by jcantrill** [comment]
@benjaminapetersen you can test by manually editing a namespace to add the appropriate annotation 'oc edit ns <NSNAME>'

**Comment by benjaminapetersen** [comment]
&& see the query working in kibana?

**Comment by jcantrill** [comment]
@benjaminapetersen you should be directed to kibana with the appropriate query working yes

**Comment by jcantrill** [comment]
@spadgett Can i straight cherry-pick this to resolve https://bugzilla.redhat.com/show_bug.cgi?id=1547348

**Comment by spadgett** [comment]
Should be able to. You might need to build dist if it doesn’t cherry pick cleanly

**Pull request merged**